src/pyhf/infer/test_statistics.py

```diff
@@ -43,7 +43,7 @@ def _tmu_like(
     )
     log_likelihood_ratio = fixed_poi_fit_lhood_val - unconstrained_fit_lhood_val
     tmu_like_stat = tensorlib.astensor(
-        tensorlib.clip(log_likelihood_ratio, 0.0, max_value=None)
+        tensorlib.clip(2.0 * log_likelihood_ratio, 0.0, max_value=None)
```

Comment:
why is this changed?

Reply:
Because the test statistic that we use should still be based on `-2*Lambda`, but the `2` is no longer automatically included in the `fixed_poi_fit_lhood_val` and `unconstrained_fit_lhood_val` above.